packages/graphql-server/src/utils/roles.ts

```diff
@@ -19,3 +27,11 @@ export const isClient = rule({ cache: 'contextual' })(
     return ctx.role === Role.CLIENT;
   },
 );
+
+export const isGroupAdmin = rule({ cache: 'contextual' })(
```

Comment:
Is `isGroupAdmin` still needed? I think all the ABAC should be all in the resolver.